README.md

```diff
@@ -55,9 +55,9 @@ This assumes `python` is the command configured to run the correct python versio
 
 ### Set up the "secret key" used by Django to secure forms.
 
-- Set up an environment variable `DJANGO_SECRET_KEY` to whatever you want to set it to.
+- Set up an environment variable `DJANGO_SECRET_KEY_PDX` to whatever you want to set it to.
```

Comment:
Should these actually be the `_PDX` versions or just `DJANGO_SECRET_KEY`?

Reply:
WHOOPS thanks.